build.ps1

```diff
@@ -775,6 +822,8 @@ function Build-Compilers($Arch, [switch]$Test = $false) {
         # give us a sligtly faster build.
         CMAKE_BUILD_TYPE = "Release";
         CMAKE_INSTALL_PREFIX = "$($Arch.ToolchainInstallRoot)\usr";
+        CMAKE_Swift_COMPILER = "$BinaryCache\toolchains\$PinnedToolchain\PFiles64\Swift\Toolchains\0.0.0+Asserts\usr\bin\swiftc.exe";
+        CMAKE_Swift_FLAGS = "-sdk $BinaryCache\toolchains\$PinnedToolchain\PFiles64\Swift\Platforms\Windows.platform\Developer\SDKs\Windows.sdk";
```

Comment:
@("-sdk", "$BinaryCache\toolchains\$PinnedToolchain\PFiles64\Swift\Platforms\Windows.platform\Developer\SDKs\Windows.sdk")

We need this so we can escape spaces properly (ie within one of the $Vars but not between -sdk and the path)

Reply:
Ugh, good point!